documentation/docs/quickstart/2-install.md

```diff
@@ -1,85 +1,37 @@
-
-![helper_scripts](/assets/img/helper_scripts.gif)
-
-Ensure the scripts have been downloaded by running `ls -l` command.
-
-![list_scripts](/assets/img/list_scripts.png)
-
-
-### Step 2: Install Docker
 
-Docker is an open source containerization product that pre-packages all dependencies into a single container, greatly simplifying the installation process. Execute the command below to install Docker using the script we downloaded in the previous step.
+## macOS
 
-```bash
-./install-docker-ubuntu.sh
-```
+  <iframe width="616" height="347" src="https://www.youtube.com/embed/klN-ToclwW4" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen>
+  </iframe>
 
-Close and restart your bash/terminal window to enable the correct permissions for the `docker` command. Otherwise, you may encounter an error in the next steps.
+Step 1. Download Hummingbot from our official website 
 
-We also have instructions for installing Docker on [Debian](/installation/via-docker/linux/#debian) and [CentOS](/installation/via-docker/linux/#centos).
-
-
-### Step 3: Install Hummingbot
-
-Execute `create.sh` script which will help you install an instance of Hummingbot and set up folders to house your logs, configuration files and trades/orders database file.
-
-```
-hummingbot_files       # Default name of top level folder for hummingbot-related files
-├── hummingbot_conf    # Maps to hummingbot's conf/ folder, which stores configuration files
-├── hummingbot_logs    # Maps to hummingbot's logs/ folder, which stores log files
-└── hummingbot_data    # Maps to hummingbot's data/ folder, which stores the SQLite database file
-```
-
-Run command to execute the script.
-
-```
-./create.sh
-```
-
-Follow the prompts to choose a version, give your Hummingbot instance a name, and enter the name of the folder where its files will be saved.
+- [Download the Hummingbot client](https://hummingbot.io/download/)
 
-![script_create](/assets/img/script_create1.gif)
+Step 2. Install Hummingbot on your local computer
 
-Afterwards, you should see the Hummingbot client interface. Proceed to [Configure a Bot](/quickstart/3-configure-bot).
+- [macOS Binary Installation](/installation/from-binary/macos)
 
+## Windows
 
-## Windows and macOS
+  <iframe width="616" height="347" src="https://www.youtube.com/embed/9TsZ_xjExXs"    frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen>
+  </iframe>
 
-Setup and install package to install on local computer can be downloaded from our official website:
+Step 1. Download Hummingbot from our official website 
 
 - [Download the Hummingbot client](https://hummingbot.io/download/)
 
-You may also refer to our User Manual for more information.
+Step 2. Install Hummingbot on your local computer
 
 - [Windows Binary Installation](/installation/from-binary/windows)
-- [macOS Binary Installation](/installation/from-binary/macos)
 
+## Linux
```

Comment:
Combine this section with **Hummingbot for Developers** and call the combined section **Linux and Advanced Users**. I would link to the following articles in this section:
* Set up cloud server: http://localhost:8000/installation/cloud/
* Install via Docker (Linux): http://localhost:8000/installation/via-docker/linux/
* Install via Source (Linux): http://localhost:8000/installation/from-source/linux/